### ISSUE TITLE
`moar` does not print a trailing newline in `-quit-if-one-screen` mode

### ISSUE THREAD

**Issue by ilyagr** [walles/moar#165]
```shell
$ echo aa |MOAR= moar -quit-if-one-screen && echo bb
aabb
$ moar --version
v1.18.3
```

The expected result of the first command would have been:

```
aa
bb
```

I've also seen this bug in previous versions of `moar`.

This is on Linux, Debian testing (trixie)

**Comment by walles** [comment]
Fixed in 1.18.4: https://github.com/walles/moar/releases/tag/v1.18.4

Thanks for the super clear report @ilyagr!

**Comment by ilyagr** [comment]
Thank you very much!
